doc/src/manual/strings.md

```diff
@@ -150,14 +150,14 @@ julia> 'x' - 'a'
 23
 
 julia> 'A' + 1
-'B'
+'B': ASCII/Unicode U+0042 (category Lu: Letter, uppercase)
 ```
 
 ## String Basics
 
 String literals are delimited by double quotes or triple double quotes:
 
-```julia
+```jldoctest
```

Comment:
Yes, found out about named blocks after submitting this PR, hence https://github.com/JuliaDocs/Documenter.jl/pull/413 :)

I will update to named blocks instead.